src/services/member/schemas.ts

```diff
@@ -121,6 +121,54 @@ export const getStorage: FastifySchema = {
   },
 };
 
+export const getStorageFiles: FastifySchema = {
+  querystring: {
+    type: 'object',
+    properties: {
+      page: { type: 'number', default: 1 },
+      pageSize: { type: 'number', default: 10 },
```

Comment:
The default here is not the same as in `src/services/member/constants.ts`? Should we use the constants here?